javascript/builtins/Intl/NumberFormat.json

```diff
@@ -962,9 +962,9 @@
                 "deprecated": false
               }
             },
-            "result_useGrouping_property": {
+            "result_number_v3_properties": {
               "__compat": {
-                "description": "The result <code>useGrouping</code> parameter may contain <code>'always'</code>, <code>'auto'</code>, <code>'min2'</code>, <code>true</code> and <code>false</code> (previous versions may only contain <code>true</code> and <code>false</code>)",
+                "description": "Intl.NumberFormat V3 proposal properties are supported: <code>roundingMode</code>, <code>roundingIncrement</code>, <code>roundingPriority</code>, and <code>trailingZeroDisplay</code>. The result <code>useGrouping</code> property may contain <code>'always'</code>, <code>'auto'</code>, <code>'min2'</code>, <code>true</code> and <code>false</code> (previous versions may only contain <code>true</code> and <code>false</code>). The result <code>signDisplay</code> property may contain <code>'negative'</code>.",
```

Comment:
re "negative" - like useGrouping, it returns other values, but now it adds `negative`.